firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -15,13 +15,18 @@
 package com.google.firebase.installations;
 
 import android.text.TextUtils;
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import com.google.firebase.installations.local.PersistedInstallationEntry;
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
 
-/** Util methods used for {@link FirebaseInstallations} */
-class Utils {
+/**
+ * Util methods used for {@link FirebaseInstallations}.
+ *
+ * @hide
```

Comment:
out of curiosity:
what does this mean/do?

Reply:
When applied to a package, class, method or field, @hide removes that node and all of its children from the public documentation.